atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/keyvalue/TableMappingService.java

```diff
@@ -27,5 +27,5 @@
     TableReference getMappedTableName(TableReference tableRef) throws TableMappingNotFoundException;
     <T> Map<TableReference, T> mapToShortTableNames(Map<TableReference, T> tableMap)
             throws TableMappingNotFoundException;
-    Set<TableReference> mapToFullTableNames(Set<TableReference> tableNames);
+    Map<TableReference, TableReference> generateMapToFullTableNames(Set<TableReference> tableNames);
```

Comment:
The large internal product will have to change it in one cli and a pair of tests, but otherwise should be straightforward.  I've updated the release notes to reflect the API break.